hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/configuration/RestConfiguration.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
I wanted to preserve only 1 config file (the `hawkular-inventory.properties`). The issue with this is that it loads all the key-value pairs from it and pass it to the Titan. But Titan complains about keys, it doesn't know, hence the `rest.` prefix and filtering it out from the implementation configuration. To do that the api module needs to know that it should filter out the `rest.` entries -> RestConfiguration class is in the api module :/

So what do you suggest? I am +1 on keeping things simple rather than introducing a new config file / config handle mechanism for each layer. But having the config for rest is quite useful, ok what about creating a dedicated config for rest using Apache Commons Configuration?